### ISSUE TITLE
Stuck with no lyrics from songmeanings (asking to create account)

### ISSUE THREAD

**Issue by JF10R** [SimonIT/spotifylyrics#26]
I keep getting no lyrics, but an invitation to create an account or login at Songmeanings
![image](https://user-images.githubusercontent.com/2191497/86131257-68e56d80-bab3-11ea-8cd2-6cf88257f2f8.png)

**Comment by JF10R** [comment]
To add onto that, the 1.21 build uses wikia as the lyrics source and displays them correctly, if it can help.

![image](https://user-images.githubusercontent.com/2191497/86131413-a0541a00-bab3-11ea-98bf-2d03060751bf.png)

**Comment by Arecsu** [comment]
I have the same problem.
Managed to solve it fixing ~4 lines in services.py related to SongMeanings

### Buggy
```
lis = soup.find_all("li")
if len(lis) > 4:
   temp_lyrics = lis[4]
   lyrics = temp_lyrics.getText()
   lyrics = lyrics.split("(r,s)};})();")[1]
```

### Fixed
```
lis = soup.find_all('ul', attrs={'data-inset' : True})
temp_lyrics = lis[1].find_all('li')[1]
lyrics = temp_lyrics.getText()
#lyrics = lyrics.split("(r,s)};})();")[1]
```

Don't have to pull request this, so I'm leaving it there @SimonIT 😁

**Comment by SimonIT** [comment]
Fixed with #27
